assets/js/blocks/reviews-by-product/style.scss

```diff
@@ -34,121 +35,139 @@
 	}
 
 	.wc-block-reviews-by-product__orderby {
-		margin-bottom: $gap-larger;
+		margin-bottom: $gap-small;
 		text-align: right;
 	}
 
 	.wc-block-reviews-by-product__orderby__label {
 		margin-right: $gap-small;
+		display: inline-block;
+		font-weight: normal;
```

Comment:
Are these two rules required for a specific browser? They don't seem to make any difference for me.

Reply:
For specific themes. Some of the 20x themes need this.

Reply:
> For specific themes. Some of the 20x themes need this.

If a theme adds a style to `label` elements, shouldn't we honor those styles here too? Otherwise it sounds to me like it's a bug in the theme and not in our block: a theme shouldn't add styles to HTML elements if not all instances of that element are going to use those styles.

What worries me about that is that having to reset browser styles for certain HTML tags sounds like a rabbit hole. What if a theme did `div { display: inline }`, would we have to set `display: block` in every div we use in WooCommerce? :confused:

Reply:
In this case the theme added display block. We should offer basic compatibility with basic themes - we cannot really expect the 20x themes to account for our custom markup.